my-app/src/app/courses/course-editor/course-editor.component.html

```diff
@@ -61,7 +62,7 @@
           <button class="btn btn-danger"
             (click)="cancel()" id="cancel-button">Cancel</button>
           <button class="btn btn-primary offset-1"
-            (click)="save()" id="delete-button">Save</button>
+            (click)="isNew ? save() : update()" id="delete-button">Save</button>
         </div>
       </div>
 
```

Comment:
For creation date field it would be better to use material date picker
https://material.angular.io/components/datepicker/overview